src/components/snackbar/snackbar.tsx

```diff
@@ -183,27 +183,27 @@ export class Snackbar {
 
         return (
             <div class="mdc-snackbar__actions" aria-atomic="true">
-                {this.renderActionButton(actionText)}
-                {this.renderDismissButton(dismissible)}
+                {this.renderActionButton()}
+                {this.renderDismissButton()}
```

Comment:
This is more a matter of style, some programmers prefer to expose all the used parameters in a function whereas others encapsulate the behaviour regardless. But I think if we change it here, it would be more consistent to do the same on line 180 and not pass anything into renderActions()

Reply:
I think in all of our newest components, we do as I did now in this refactor. That should mean our team prefers this stye 😄  I feel the other one requires a lot of bookkeeping and remembering stuff